airbase/pom.xml

```diff
@@ -1126,18 +1127,16 @@
             </dependency>
 
             <!-- misc stuff -->
-
-            <!-- bval 1.1.1 depends on commons-lang3 3.3.2, which is incompatible with Java 9 -->
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-lang3</artifactId>
-                <version>3.4</version>
+                <version>3.7</version>
             </dependency>
 
             <dependency>
                 <groupId>org.apache.bval</groupId>
                 <artifactId>bval-jsr</artifactId>
-                <version>1.1.1</version>
+                <version>${dep.bval.version}</version>
                 <exclusions>
```

Comment:
None of these exclusions should be needed now. I added it to a new project and `dependency:tree` shows no dependencies. Although it's strange that `commons-weaver-privilizer-api` is declared as a dependency but Maven doesn't pick it up.

Reply:
Ah, I see it's not picked up because the parent POM declares it as `provided` scope in `dependencyManagement`. I didn't realize you could do that (and it seems strange).